src/mimefactory.rs

```diff
@@ -2084,4 +2084,33 @@ mod tests {
 
         Ok(())
     }
+
+    /// Tests that From: header comes before Autocrypt:
+    ///
+    /// Autocrypt headers are usually large, so this supposedly
+    /// makes Delta Chat messages more robust against buggy parsers that
+    /// can't read past Autocrypt header.
+    ///
+    /// It's unclear whether such parsers actually exist, the test is added as a precaution.
```

Comment:
as this is now shown to be not true i'd just say it's to keep to the convention of starting with from

Reply:
Removed the comment.